src/Post/Post.js

```diff
@@ -161,21 +182,20 @@ function CommentReplySection({ comment }) {
             <p className="replier-username-text">
               <strong>{reply.user_name}</strong> {reply.comment}
             </p>
-            <svg
-              aria-label="Like"
-              className="reply-like-icon"
-              color={reply.is_liked_by_user ? '#ed4956' : '#262626'}
-              height="12"
-              role="img"
-              viewBox="0 0 48 48"
+            <PostLikeButton
+              index={postIndex}
+              is_post_liked={reply.is_liked_by_user}
+              size="12"
               width="12"
-            >
-              <path d="M34.6 6.1c5.7 0 10.4 5.2 10.4 11.5 0 6.8-5.9 11-11.5 16S25 41.3 24 41.9c-1.1-.7-4.7-4-9.5-8.3-5.7-5-11.5-9.2-11.5-16C3 11.3 7.7 6.1 13.4 6.1c4.2 0 6.5 2 8.1 4.3 1.9 2.6 2.2 3.9 2.5 3.9.3 0 .6-1.3 2.5-3.9 1.6-2.3 3.9-4.3 8.1-4.3m0-3c-4.5 0-7.9 1.8-10.6 5.6-2.7-3.7-6.1-5.5-10.6-5.5C6 3.1 0 9.6 0 17.6c0 7.3 5.4 12 10.6 16.5.6.5 1.3 1.1 1.9 1.7l2.3 2c4.4 3.9 6.6 5.9 7.6 6.5.5.3 1.1.5 1.6.5.6 0 1.1-.2 1.6-.5 1-.6 2.8-2.2 7.8-6.8l2-1.8c.7-.6 1.3-1.2 2-1.7C42.7 29.6 48 25 48 17.6c0-8-6-14.5-13.4-14.5z"></path>
-            </svg>
+              toggleLike={toggleCommentReplyLike}
+              commentId={comment.comment_id}
+              replyId={reply.comment_id}
+              isMarginNeeded="-180px"
```

Comment:
This is quite dirty. Try to avoid [magicn umbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) as much as you can, later you'll come to this and think wtf is 180?. Also, when we want to turn this responsive, all magic numbers will bite us in the ass.

We need this due to a design flaw, we put the replies inside the comment body, so they couldn't take the full width. I think you have to take them out of the `comment-body` div and instead make them siblings like so
```jsx
<div className="comment-wrapper" />
<div className="comment-action-container" />
<div className="view-replies-section" />
```
This way everyone can take full width.